### PR TITLE
feature(component): info panel

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: [#425](https://app.zenhub.com/workspaces/ffam-prototype-5c615588801d5a103031639c/issues/unimelb/ffam-prototype/425)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 077f05a23ac81f2af69f737343da026f6f6516a4

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5ceb41da9907cf000985337b

File: components/info-panel/InfoPanel.vue [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <SvgIcon
+        :name="iconName"
+        :aria-label="iconName"
+        class="info-panel__icon"
+        width="20px"
+        height="20px"
+      />
+    </div>
+    <div class="info-panel__container">
+      <h6 class="title--xxs info-panel__title">{{ title }}</h6>
```
Comment (TimMansell) [inline components/info-panel/InfoPanel.vue]: I think an h4 would be better suited for this.

File: components/info-panel/InfoPanel.vue [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <div>
+      <SvgIcon
+        :name="iconName"
+        :aria-label="iconName"
+        class="info-panel__icon"
+        width="20px"
+        height="20px"
+      />
+    </div>
+    <div class="info-panel__container">
```
Comment (TimMansell) [inline components/info-panel/InfoPanel.vue]: Rename this to `info-panel__inner` to match other components structure.

File: components/info-panel/stories/Story1.vue [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                width="18px"
+                height="18px"/>
+            </a>
+          </InfoPanel>
+        </ListItem>
+        <ListItem>
+          <InfoPanel
+            icon-name="location"
+            title="Location">
+            <p>Type: Concert</p>
```
Comment (TimMansell) [inline components/info-panel/stories/Story1.vue]: Use a class on the `<p>` to style

File: components/info-panel/index.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    margin-right: 1rem;
+  }
+
+  &__inner {
+    display: flex;
+    flex-direction: column;
+    align-content: flex-start;
+  }
+
+  p {
```
Comment (TimMansell) [inline components/info-panel/index.css]: convert to BEM

File: components/info-panel/index.css [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    margin-bottom: 0;
+  }
+
+  svg {
```
Comment (TimMansell) [inline components/info-panel/index.css]: move to `&__icon`

File: components/info-panel/InfoPanel.vue [hunk 6]
```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="info-panel">
+    <div>
```
Comment (TimMansell) [inline components/info-panel/InfoPanel.vue]: Don't need this `<div>`

**Comment by TimMansell** [comment]
SVG icon fill colour for external links will be addressed in #1123

**Pull request merged**